Mixpanel/Mixpanel.m

```diff
@@ -1854,8 +1852,33 @@ - (void)executeCachedEventBindings {
 
 @end
 
-#pragma mark - People
+#pragma mark - WatchExtensions
+@implementation Mixpanel (WatchExtensions)
+
+/** Called on the delegate of the receiver. Will be called on startup if the incoming message caused the receiver to launch. */
+- (void)session:(WCSession *)session didReceiveMessage:(NSDictionary<NSString *, id> *)message {
+    if ([Mixpanel isValidWatchSessionMessage:message]) {
+        [[Mixpanel sharedInstance] track:message[@"event"] properties:message[@"properties"]];
```

Comment:
We will have to parse the message[@"$mp_message_type"] type here and act accordingly (i.e. track, increment, etc..). We will only support track for now.